src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueAdapterFactory.cs

```diff
@@ -77,6 +80,13 @@ public virtual void Init(IProviderConfiguration config, string providerName, Log
                     throw new ArgumentException(String.Format("{0} invalid.  Must be int", NUM_QUEUES_PARAM));
             }
 
+            string boolString;
+            if (!config.Properties.TryGetValue(FAULT_ON_DELIVERY_FAILURE, out boolString))
```

Comment:
we don't need to allow faultOnDeliveryFailure for AQ adapter. We can always have it be false, via NoOpStreamDeliveryFailureHandler be lase by default.